### PR TITLE
Removes unused revival code from revenants

### LINKED ISSUE #12787: Revenant does not revive on death like intended.
**Issue Description**:

Revenants, if there ectoplasm is not disturbed for a minute after death, are supposed to revive from the dust. However, due to how the dust is spawned, it does not work or happen.

**What did you expect to happen**: 

To rise from the dust of your grave again

**What happened instead**:

The dust just becomes inert

**Why is this bad/What are the consequences**:

Crew should have to remove / clean up the dust to keep it from spawning

**Steps to reproduce the problem**:

Be a revenant
Die from anything
Notice after a minute the dust becomes inert.

**When did the problem start happening**:

Forever, probably

**Extra information**:

The variable reforming is what is checked to revive the ghost. For some reason, it is coded that the dust spawns set to zero... preventing the dropped dust from ever coming back to life, and instead always going inert.

### PR DESCRIPTION
**Pull request by Fluffyzack21**
## What Does This PR Do
Pretty much what the title says,

Fixed #12787 

## Why It's Good For The Game
 Can't revive a dead revenant again after it sucked enough of dead corpses

## Testing
 Spawned as the revanant and drained there essence from a bunch of dead skrells then killed the revanant and tried to revive it but can't

## Changelog
:cl:
del: removes ectoplasm scattering from revenants
/:cl:

### CODE REVIEW

**Comment by CinnamonSnowball** [comment]
This PR removes ectoplasm scattering, so changelog should probably mention it, it's not NPFC.

**Comment by Fluffyzack21** [comment]
> This PR removes ectoplasm scattering, so changelog should probably mention it, it's not NPFC.

Okay should be fixed

File: code/game/gamemodes/miniantags/revenant/revenant.dm [hunk 1]
```diff
@@ -362,83 +359,7 @@
...
-		player_mind.assigned_role = SPECIAL_ROLE_REVENANT
-		player_mind.special_role = SPECIAL_ROLE_REVENANT
-		SSticker.mode.traitors |= player_mind
-		message_admins("[key_of_revenant] has been [client_to_revive ? "re":""]made into a revenant by reforming ectoplasm.")
-		log_game("[key_of_revenant] was [client_to_revive ? "re":""]made as a revenant by reforming ectoplasm.")
-		visible_message("<span class='revenboldnotice'>[src] suddenly rises into the air before fading away.</span>")
-		qdel(src)
-		if(src) //Should never happen, but just in case
-			inert = 1
+	. += "<span class='revennotice'>It seems inert.</span>"
```
Comment (1080pCat) [inline code/game/gamemodes/miniantags/revenant/revenant.dm:365]:
```suggestion
	. += "<span class='revennotice'>Lifeless, yet sparkling ectoplasm. From what was once a spirit seeking revenge on the station.</span>"
```

File: code/game/gamemodes/miniantags/revenant/revenant.dm [hunk 2]
```diff
@@ -362,83 +359,7 @@
...
-		player_mind.assigned_role = SPECIAL_ROLE_REVENANT
-		player_mind.special_role = SPECIAL_ROLE_REVENANT
-		SSticker.mode.traitors |= player_mind
-		message_admins("[key_of_revenant] has been [client_to_revive ? "re":""]made into a revenant by reforming ectoplasm.")
-		log_game("[key_of_revenant] was [client_to_revive ? "re":""]made as a revenant by reforming ectoplasm.")
-		visible_message("<span class='revenboldnotice'>[src] suddenly rises into the air before fading away.</span>")
-		qdel(src)
-		if(src) //Should never happen, but just in case
-			inert = 1
+	. += "<span class='revennotice'>Lifeless, ectoplasm, still faintly glimmering in the light. From what was once a spirit seeking revenge on the station.</span>"
```
Comment (1080pCat) [inline code/game/gamemodes/miniantags/revenant/revenant.dm:365]:
```suggestion
	. += "<span class='revennotice'>Lifeless ectoplasm, still faintly glimmering in the light. From what was once a spirit seeking revenge on the station.</span>"
```
GUH my bad sorry

Comment (Fluffyzack21) [reply code/game/gamemodes/miniantags/revenant/revenant.dm]: its all good

**Review by Henri215 — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**